lunabot_embedded/firmware/new_encoder_test/new_encoder_test.ino

```diff
@@ -0,0 +1,29 @@
+/* Encoder Library - Basic Example
+ * http://www.pjrc.com/teensy/td_libs_Encoder.html
+ *
+ * This example code is in the public domain.
+ */
+
```

Comment:
rename this file to test_amt13_encoder. if we ever get other encoder, this code will not be testing a new encoder